host/tests/L3_TestCases/dsCompositeIn/dsCompositeInHelperClass.py

```diff
@@ -63,7 +63,9 @@ def __init__(self, testName:str, qcId:str, log:logModule=None ):
         # Get path to device profile file
         self.deviceProfile = os.path.join(dir_path, self.cpe.get("test").get("profile"))
 
-        self.deviceDownloadPath = self.cpe.get("target_directory")
+        self.targetWorkspace = self.cpe.get("target_directory")
+        self.targetWorkspace = os.path.join(self.targetWorkspace, self.moduleName)
+        self.streamDownloadURL = self.cpe.get("test").get("streams_download_url")
 
     def testDownloadAssets(self):

```

Comment:
do we required this function? as there is no stream to download

Reply:
streams not required so removed this